base/namedtuple.jl

```diff
@@ -280,6 +280,16 @@ get(f::Callable, nt::NamedTuple, key::Union{Integer, Symbol}) = haskey(nt, key)
     (names...,)
 end
 
+"""
+    delete(nt::NamedTuple{an}, omitnames::Tuple{Vararg{Symbol}}) where {an}
```

Comment:
@JeffreySarnoff do you think it is really necessary to give the parametric type in the named tuple? I don't think the `{an} where {an}` adds something and makes the docstring longer and less readable.

Reply:
I copied the form used in another NamedTuple function.  The `an` captures the names directly, rather than requiring a separate function call.

Reply:
I am referring to the _documentation string_. Should the user care about this implementation detail? Isn't it anyway clear that a named tuple is parametricaly defined?

Reply:
really, though for the docstring could be given without that and still be valid -- just not matching
I have no idea if that is frowned upon.
```
"""
    delete(nt::NamedTuple, omitnames::Tuple{Varag{Symbol}})
"""

```

Reply:
I will shorten the doc string -- and if there are voices to put it back -- I will do that.